recipes/nvmolkit/recipe.yaml

```
@@ -0,0 +1,67 @@
+context:
+  version: "0.2.0"
+
+package:
+  name: "nvmolkit"
+  version: ${{ version }}
+
+source:
+  url: https://github.com/NVIDIA-Digital-Bio/nvMolKit/archive/v${{ version }}.tar.gz
+  sha256: 8c0181c4dcd02aac42dc3c1b783ac677a8149fd5d5b5d25e85ba30e85bd6ec5a
+
+build:
+  number: 0
+  skip:
+    - osx
+    - win
+    - cuda_compiler == 'None'
+requirements:
+  build:
+    - ${{ stdlib('c') }}
+    - ${{ compiler('c') }}
+    - ${{ compiler('cxx') }}
+    - ${{ compiler('cuda') }}
+    - cmake >=3.30
+    - make
+  host:
+    - cuda-runtime
+    - cuda-version >=12.6
```

For `cuda-version`, we can align the `host` requirement like so

```suggestion
    - cuda-version {{ cuda_compiler_version }}
```